be/src/Project/Web/Seller/Seller.Web/Areas/Products/ApiRequestModels/SaveProductRequestModel.cs

```diff
@@ -17,6 +17,7 @@ public class SaveProductRequestModel : RequestModelBase
         public IEnumerable<SaveFileRequestModel> Files { get; set; }
         public bool IsNew { get; set; }
         public string Ean { get; set; }
+        public int FulfillmentTime { get; set; }
```

Comment:
**suggestion:** Let's make it nullable and optional on the product card. It should put null into the database if no fulfillment time has been set